bin/legacypipe-env

```diff
@@ -0,0 +1,39 @@
+## bashrc snippet for setting up environment for runbrick.py
+
+unset PYTHONPATH
+
+export NERSC_HOST=`/usr/common/usg/bin/nersc_host`
+
+# Where's the code?
+# export LEGACY_PIPE_DIR=/global/cscratch1/sd/desiproc/${NERSC_HOST}-code/legacypipe/
+
+# Where are inputs and outputs going?
+export LEGACY_SURVEY_DIR=/global/cscratch1/sd/desiproc/dr7
+
+export DUST_DIR=/global/cscratch1/sd/desiproc/dust/v0_0
+
+export UNWISE_COADDS_DIR=/global/projecta/projectdirs/cosmo/work/wise/outputs/merge/neo4/fulldepth:/global/project/projectdirs/cosmo/data/unwise/allwise/unwise-coadds/fulldepth
+
+export UNWISE_COADDS_TIMERESOLVED_DIR=/global/projecta/projectdirs/cosmo/work/wise/outputs/merge/neo3
+
```

Comment:
The NEO4 time-resolved coadds are now ready, so this variable (don't we need the index, too?) should be
```
export UNWISE_COADDS_TIMERESOLVED=/global/projecta/projectdirs/cosmo/work/wise/outputs/merge/neo4                              
export UNWISE_COADDS_TIMERESOLVED_INDEX=/global/projecta/projectdirs/cosmo/work/wise/outputs/merge/neo4/time_resolved_atlas.fi\
ts                                                                                                                             
```